README.md

```diff
@@ -0,0 +1,11 @@
+# practice-questions
+Practice questions for interviews
+the website where you can practice :
+1)HackerRank
+2)HackerEarth
+3)GreeksForGreek
```

Comment:
You have misspelled it. Can you please correct it?

Reply:
ok